roles/bootstrap-os/tasks/main.yml

```diff
@@ -43,6 +43,10 @@
     gather_subset: '!all'
     filter: ansible_*
 
+- name: Set OS families based on distribution
+  set_fact:
+    os_families: "{{ 'openEuler' if ansible_distribution == 'openEuler' else ansible_os_family }}"
+
```

Comment:
Thank you for reaching out.

Since openEuler is very *similar* to Red Hat-based distributions, modifying the os_families in Kubespray might not be the most efficient solution. Changing os_families could have widespread implications, as it affects how Kubespray determines which tasks to run based on the operating system family.

It's better if a lighter way to solve the issue is to adjust how Ansible recognizes openEuler without changing the `os_families` globally.

Reply:
HI @ErikJiang

Would you please give us some comments :-)